website/routes.py

```diff
@@ -430,13 +441,6 @@ def make_url_map(app):
             OsfWebRenderer('public/pages/preprint_landing.mako', trust=False),
         ),
 
-        Rule(
-            '/registries/',
```

Comment:
We need to retain compatibility for user-facing routes, so this should not be removed.

Reply:
I think this is redirected at a higher level, even when this is removed the user is still sent to the Ember app OSF Registries landing page. So this doesn't seem to do anything.